gitignore

From: Bryan Larsen <bryan@larsen.st>

### Housekeeping

Because this is a gitorial, we will sometimes have to create steps
that you can ignore.  In this case, we're creating a .gitignore to
make our life easier.  You can do so, or not, as you wish.
---

 .gitignore |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)
 create mode 100644 .gitignore


diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..7b84d97
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,7 @@
+tmp/*
+log/*.log
+db/*.sqlite3
+app/views/taglibs/auto
+db/data.yml
+patches-*/*
+.stgit-edit.txt
